### PR TITLE
feat: send message activities to studio

### PR DESCRIPTION
**Pull request by rabe1028**
## Description

We want to send message activities to Studio when the api `create-verifiable-message` `verify-verifiable-message` called.

I also added a test case for UseCase failure.

## Check 

- [x] E2E Test only
  - This is because these APIs in the studio are still being implemented.

### CODE REVIEW

File: src/nodex/utils/hub_client.rs [hunk 1]
```diff
@@ -180,11 +180,16 @@ impl HubClient {
         self.post(path, &payload).await
     }
 
-    #[allow(dead_code)]
-    pub async fn put(&self, _path: &str) -> anyhow::Result<reqwest::Response> {
-        let url = self.base_url.join(_path)?;
+    pub async fn put(&self, path: &str, body: &str) -> anyhow::Result<reqwest::Response> {
```
Comment (rabe1028) [inline src/nodex/utils/hub_client.rs:183]: To send a body, I added new argument `body`

File: src/controllers/public/nodex_create_verifiable_message.rs [hunk 2]
```diff
@@ -16,6 +16,7 @@ use crate::{
 pub struct MessageContainer {
     destination_did: String,
     message: String,
+    operation_tag: String,
```
Comment (rabe1028) [inline src/controllers/public/nodex_create_verifiable_message.rs:19]: I forgot to add `operation_tag`.

File: src/repository/message_activity_repository.rs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    pub message_id: Uuid,
+    pub from: String,
+    pub to: String,
+    pub operation_tag: String,
+    pub is_encrypted: bool,
+    pub occurred_at: DateTime<Utc>,
+}
+
+#[derive(Clone, Debug, Serialize)]
+pub struct VerifiedMessageActivityRequest {
```
Comment (daichitominaga) [inline src/repository/message_activity_repository.rs:16]: For the purpose of verifying Device and MessageActivity, I have added `to` and `from` to the body. As a result, these have also become necessary in the agent's body.

https://github.com/nodecross/nodex-platform/pull/136/commits/cd3e0f67d179feb9ca62dee80ae63ba853f12ea1

Comment (rabe1028) [reply src/repository/message_activity_repository.rs:16]: I fixed it in this commit [a74a844](https://github.com/nodecross/nodex/pull/241/commits/a74a844c39db9e907596f45029423e5e8fb9ce3e)

**Review by yokishava — APPROVED** [review]
LGTM!

**Pull request merged**